lib/qiita/markdown/filters/external_link.rb

```diff
@@ -0,0 +1,35 @@
+module Qiita
+  module Markdown
+    module Filters
+      class ExternalLink < HTML::Pipeline::Filter
+        def call
+          return doc unless hostname
```

Comment:
@yujinakayama I see. Ok, I will add the `validate` method to check whether the `hostname` is passed or not so that the `hostname` parameter is required for the ExternalLink.

Sorry, I have one more question. Actually, in the result of the discussion above, should we NOT add the `ExternalLink` to the `DEFAULT_FILTERS` defined at `lib/qiita/markdown/processor.rb` so that all qiita-markdown users can continue to use qiita-markdown without specifying `hostname` parameter?